app/sqaws.py

```diff
@@ -130,6 +131,17 @@ def build_instance_model(pricing: PricingData, region_name: str, instance_dict:
                     nagbot_state=nagbot_state)
 
 
+# Find 'stop after' and 'terminate after' fields in an EC2 instance, regardless of formatting
+def get_stop_and_terminate_after(tags: dict) -> list:
+    stop_terminate = ['', '']
```

Comment:
(style nit) Python lets you return multiple values (technically it is returning a tuple); I think it would be more idiomatic python to do 

```
def get_stop_and_terminate_after(tags: dict) -> Tuple[String, String]:
    stop_after = ''
    terminate_after = ''
    # ...
    return stop_after, terminate_after
```

As a bonus you can still initialize them both on the same line like `stop_after, terminate_after = '', ''`